ansible/roles/infra-osp-template-generate/defaults/main.yaml

```diff
@@ -8,3 +8,71 @@ opentlc_admin_pub_keys:
 all_ssh_authorized_keys: "{{ opentlc_admin_pub_keys + [user_pub_key|d()] }}"
 
 provider_network: external
+
+default_security_groups:
+  - name: BastionSG
+    description: >-
+      Bastion security group allows basic ICMP
+      and SSH adn Mosh ingress and egress to *
+    rules:
+      - name: MoshPublic
+        description: "Public Mosh Access for bastions"
+        from_port: 60000
+        to_port: 61000
+        protocol: udp
+        cidr: "0.0.0.0/0"
+        rule_type: Ingress
+
+      - name: SSHPublic
+        description: "Public Access for bastions"
+        from_port: 22
+        to_port: 22
+        protocol: tcp
+        cidr: "0.0.0.0/0"
+        rule_type: Ingress
+
+  - name: DefaultSG
```

Comment:
Instead of creating another SG, why don't we just use the default one that comes automatically when we provision a new project & network? We'd just need to add a task that changes the inbound from being limited to a remote SG to 0.0.0.0/0

Reply:
That way it's explicite in agnosticd ?

Reply:
What exactly does that buy you? is there a reason that having it explicit in agnosticd is useful? 

You have a `default` SG 100% of the time in OpenStack. This would just create another `DefaultSG` SG that is intended to do the same thing. Less is more IMO, but if you really feel another SG is warranted, I am not going to stop it.

Reply:
i'm not a big fan of adding ansible tasks to just add rules in the default SG.
Here it's very self-documenting, and it's just configuration.
No more code i mean.

Unless we can just rename DefaultSG with 'default' and the heat template will update the default SG without complaining ?
I can try.

Reply:
Did you get a chance to try with this? If not, I suggest we move forward as is. While we don't need another SG, I don't think it will hurt at this point and with the new purge tasks, anything created with our outside of Heat will get cleaned up prior to project deletion.

It is always something we can iterate on later since it would be "more right" to just use the default SG that already exists.

Reply:
I haven't tried it yet.
I agree, let's move this task outside of this PR.